superdesk/sams/assets.py

```diff
@@ -0,0 +1,125 @@
+# -*- coding: utf-8; -*-
+#
+# This file is part of Superdesk.
+#
+# Copyright 2013-2018 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+
+"""The Assets API to retrieve, create, update, delete the list of ``Assets``.
+
+
+=====================   =================================================
+**endpoint name**        'assets'
+**resource title**       'Assets'
+**resource url**         [GET] '/sams/assets'
+**item url**             [GET] '/sams/assets/<:class:`str`>'
+**schema**               :attr:`sams_client.schemas.ASSET_SCHEMA`
+=====================   =================================================
+"""
+
+import logging
+import magic
+import superdesk
+from flask import request, current_app as app
+from io import BytesIO
+from superdesk.errors import SuperdeskApiError
+from werkzeug.wsgi import wrap_file
+
+logger = logging.getLogger(__name__)
+assets_bp = superdesk.Blueprint('sams_assets', __name__)
+
+
+@assets_bp.route('/sams/assets', methods=['GET'])
+def get():
+    """
+    Returns a list of all the registered assets
+    """
+    assets = assets_bp.kwargs['client'].assets.search()
```

Comment:
We would need to fill in the parameters as passed in from the client (url params)

Reply:
Resolved